### PR TITLE
feat: update global font settings

### PR DESCRIPTION
**Pull request by zhixzhan**
## Description

update default font settings align with vscode

##### Checklist

- [x] LG\LU\Qna hosit editor, 
- [x] LG\LU\Qna all up editor, 
- [x] Dialog Json editor, 
- [x] Webchat inspector

Platform: 

- [x] Electron 
- [x] Win
- [x] Mac
- [x] Linux@Shuai

## Task Item
#minor 

## Screenshots

- Electron (Mac)
image

- Windows
image

- Mac
image
image

- Linux
image

### CODE REVIEW

**Comment by hatpick** [comment]
Windows does not have these fonts down to "Courier New"

**Comment by zhixzhan** [comment]
> Windows does not have these fonts down to "Courier New"

Should we include other font as downgrade options in windows, @mewa1024 any suggestions?

**Comment by srinaath** [comment]
@zhixzhan  lets sync on it this evening.

const DEFAULT\_WINDOWS\_FONT\_FAMILY = 'Consolas, \\'Courier New\\', monospace';

const DEFAULT\_MAC\_FONT\_FAMILY = 'Menlo, Monaco, \\'Courier New\\', monospace';

const DEFAULT\_LINUX\_FONT\_FAMILY = '\\'Droid Sans Mono\\', \\'monospace\\', monospace, \\'Droid Sans Fallback\\'';

And the font weights are normal.

[vscode/editorOptions.ts at main · microsoft/vscode (github.com)](https://github.com/microsoft/vscode/blob/main/src/vs/editor/common/config/editorOptions.ts#L3680)

**Comment by mewa1024** [comment]
Let's use 400 instead of 500 (normal instead of bold).

**Review by srinaath — CHANGES_REQUESTED** [review]
Reviewing changes across platforms

File: Composer/packages/client/src/recoilModel/utils/index.ts [hunk 1]
```diff
@@ -7,6 +7,53 @@ import merge from 'lodash/merge';
...
+const DEFAULT_LINUX_FONT_FAMILY = "'Droid Sans Mono', 'monospace', monospace, 'Droid Sans Fallback'";
+
+export const DEFAULT_FONT_SETTINGS = {
+  fontFamily: PLATFORM.isMacintosh
+    ? DEFAULT_MAC_FONT_FAMILY
+    : PLATFORM.isLinux
+    ? DEFAULT_LINUX_FONT_FAMILY
+    : DEFAULT_WINDOWS_FONT_FAMILY,
+  fontWeight: 'normal',
+  fontSize: PLATFORM.isMacintosh ? '12px' : '14px',
```
Comment (hatpick) [inline Composer/packages/client/src/recoilModel/utils/index.ts:52]: use 14 for all.

File: Composer/packages/client/src/recoilModel/utils/index.ts [hunk 2]
```diff
@@ -7,6 +7,53 @@ import merge from 'lodash/merge';
 import storage from '../../utils/storage';
 import { isElectron } from '../../utils/electronUtil';
 
+const getOS = () => {
```
Comment (hatpick) [inline Composer/packages/client/src/recoilModel/utils/index.ts:10]: I suggest moving all of these to a file called ```fontUtils.ts```, then export a function called ```getDefaultFontSettings``` to get the default settings.
Also, you can check if we are on electron, you can just use ```platform.os``` to figure our the os.

File: Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/WebchatLog/WebchatLogItem.tsx [hunk 3]
```diff
@@ -13,7 +15,12 @@ export interface WebchatLogItemProps {
 export const WebchatLogItem: React.FC<WebchatLogItemProps> = ({ item }) => {
   return (
     <div
-      css={{ padding: '8px 0px', fontSize: FontSizes.size14, fontFamily: 'Consolas', color: `${NeutralColors.black}` }}
+      css={{
+        padding: '8px 0px',
+        fontSize: DEFAULT_FONT_SETTINGS.fontSize,
+        fontFamily: DEFAULT_FONT_SETTINGS.fontFamily,
+        color: `${NeutralColors.black}`,
```
Comment (hatpick) [inline Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/WebchatLog/WebchatLogItem.tsx:22]: I dont think this is necessary

**Comment by cosmicshuai** [comment]
Screenshot from 2021-02-25 10-54-23

**Review by hatpick — APPROVED** [review]
Looks good to me. Great work @zhixzhan

**Review by srinaath — APPROVED** [review]
Awesome work @zhixzhan

**Pull request merged**